controllers/updates/views/listing.njk

```diff
@@ -0,0 +1,82 @@
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+{% from "components/news.njk" import updateTeaser with context %}
+{% from "components/split-nav/macro.njk" import splitNav %}
+
+{% set bodyClass = 'has-static-header' %}
+{% extends "layouts/main.njk" %}
+
+{% block content %}
+    {% set pageAccent = 'cyan' %}
+
+    <main role="main" id="content">
+        <section class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+            {{ breadcrumbTrail(breadcrumbs) }}
+
+            <div class="{% if not isNewsLandingPage %}u-constrained-wide{% endif %}">
```

Comment:
Not a big fan of this `isNewsLandingPage` flag, would rather render two separate templates. I think they are distinct enough to warrant it.